faster_whisper_info.txt

```diff
@@ -0,0 +1,10 @@
+https://github.com/SYSTRAN/faster-whisper?tab=readme-ov-file#install-with-pip-linux-only
+
+The path can be found with
```

Comment:
@ad-astra-video this was a node for myself to remember what I did to make it work. We can remove this later.